bundles/org.openhab.core.model.core/src/main/java/org/openhab/core/model/core/internal/folder/FolderObserver.java

```diff
@@ -287,7 +289,8 @@ private void checkPath(final Path path, final WatchService.Kind kind) {
     }
 
     @Override
-    public void processWatchEvent(WatchService.Kind kind, Path path) {
+    public void processWatchEvent(WatchService.Kind kind, Path absolutePath) {
```

Comment:
Please use `fullPath` here (or change all to `absolutePath`, but it should be consistent with the naming in the interface).

Reply:
good suggestion! Whilst my gut feeling says it should be `absolutePath`, but it seems that `fullPath` is shorter and nicer, hence "easier" to type. I have changed them all to fullPath.